tf-modules/cumulus/archive.tf

```diff
@@ -48,7 +48,7 @@ module "archive" {
   saml_idp_login                  = var.saml_idp_login
   saml_launchpad_metadata_path    = var.saml_launchpad_metadata_path
 
-  urs_url             = "https://uat.urs.earthdata.nasa.gov"
+  urs_url             = var.urs_url
```

Comment:
note: this is changing the 'default' away from UAT.       Is that intended?   If so we need a CHANGELOG entry. 

In fact, probably need a CHANGELOG entry anyway.